### PR TITLE
ONE 10 cards

### PR DESCRIPTION
**Pull request by Simisays**
These common and uncommons are pretty straight forward.

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Bladehold War-Whip
+ManaCost:1 R W
+Types:Artifact Equipment
+K:For Mirrodin
+S:Mode$ ReduceCost | ValidCard$ Card.Other | ValidSpell$ Activated.Equip | Activator$ You | Amount$ 1 | Condition$ PlayerTurn | Description$ Equip abilities you activate of other Equipment cost {1} less to activate.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt:5]: Condition wrong?

File: forge-gui/res/cardsfolder/upcoming/mite_overseer.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Mite Overseer
+ManaCost:3 W
+Types:Creature Phyrexian Soldier
+PT:4/2
+K:First strike
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+token | AddPower$ 1 | AddKeyword$ First Strike | Condition$ PlayerTurn | Description$ As long as it's your turn, creature tokens you control get +1/+0 and have first strike.
+A:AB$ Token | Cost$ 3 WP | TokenScript$ c_1_1_a_phyrexian_mite_toxic_noblock | SpellDescription$ Create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and "This creature can't block."(Players dealt combat damage biy it also get a poison counter. {W/P} can be paid with either {W} or 2 life.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/mite_overseer.txt:7]: some typos here

File: forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Rhuk, Hexgold Nabber
+ManaCost:2 R
+Types:Legendary Creature Goblin Rebel
+PT:2/2
+K:Trample
+K:Haste
+T:Mode$ Attacks | ValidCard$ Creature.equipped+Other | Execute$ TrigAttach | TriggerDescription$ Whenever an equipped creature you control other than CARDNAME attacks or dies, you may attached all Equipment attached to that creature to NICKNAME.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt:7]: scryfall typos

File: forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt (hunk 1 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt:5]: Maybe `ValidCard$ Equipment.Other` too as well?

File: forge-gui/res/cardsfolder/upcoming/experimental_augury.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Experimental Augury
+ManaCost:1 U
+Types:Instant
+A:SP$ Dig | DigNum$ 3 | ChangeNum$ 1 | SubAbility$ DBProliferate | SpellDescription$ Look at the top three cards of your library. Put one of them into your hand and the rest on the bottom of your library in any order.
+SVar:DBProliferate:DB$ Proliferate
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/experimental_augury.txt:5]: Probably add SpellDescription with reminder text here so display matches Oracle

File: forge-gui/res/cardsfolder/upcoming/experimental_augury.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Name:Experimental Augury
+ManaCost:1 U
+Types:Instant
+A:SP$ Dig | DigNum$ 3 | ChangeNum$ 1 | SubAbility$ DBProliferate | SpellDescription$ Look at the top three cards of your library. Put one of them into your hand and the rest on the bottom of your library in any order.
+SVar:DBProliferate:DB$ Proliferate 
+DeckHints:Ability$Counters & Type$Planeswalker
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/experimental_augury.txt:6]: Isn't there a DeckHas for Proliferate stuff?

File: forge-gui/res/cardsfolder/upcoming/mite_overseer.txt (hunk 2 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/mite_overseer.txt:7]: Same typos also in Oracle line

File: forge-gui/res/cardsfolder/upcoming/mondrak_glory_dominus.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Mondrak, Glory Dominus
+ManaCost:2 W W
+Types:Legendary Creature Phyrexian Horror
+PT:4/4
+R:Event$ CreateToken | ActiveZones$ Battlefield | ValidToken$ Card.YouCtrl | ReplaceWith$ DoubleToken | Description$ If one or more tokens would be created under your control, twice that many of those tokens are created instead.
+SVar:DoubleToken:DB$ ReplaceToken | Type$ Amount | ValidCard$ Card.YouCtrl
+A:AB$ PutCounter | Cost$ 1 WP WP Sac<2/Artifact.Other;Creature.Other/other artifacts and/or creature> | Defined$ Self | CounterType$ Indestructible | CounterNum$ 1 | SpellDescription$ Put an indestructible counter on CARDNAME.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/mondrak_glory_dominus.txt:7]: does this cost look right in Card Detail, or should it end with `creatures>` ?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/mondrak_glory_dominus.txt:7]: Add the reminder text to SpellDescription... auto stack description should drop it

File: forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+T:Mode$ ChangesZone | ValidCard$ Creature.equipped+Other | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigAttach | Secondary$ True | TriggerDescription$ Whenever an equipped creature you control other than CARDNAME attacks or dies, you may attached all Equipment attached to that creature to NICKNAME.
+SVar:TrigAttach:DB$ Attach | Object$ Equipment.AttachedBy TriggeredCard | Defined$ Self | StackDescription$ None
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt:9]: StackDescription unneeded when nested in a Trigger

File: forge-gui/res/cardsfolder/upcoming/paladin_of_predation.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+Name:Paladin of Predation
+ManaCost:5 G G
+Types:Creature Phyrexian Knight
+PT:6/7
+K:Toxic:6
+S:Mode$ CantBlockBy | ValidAttacker$ Card.Self | ValidBlocker$ Creature.powerLE2 | Description$ CARDNAME can't be blocked by creatures with power 2 or less.
+Oracle:Toxic 6 (Plavers dealt combat damage by this creature also get six poison counters.)\n\nPaladin of Predation can't be blocked by creatures with power 2 or less.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/paladin_of_predation.txt:7]: `\n\n` ?

File: forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt [hunk 9]
```diff
@@ -0,0 +1,9 @@
+Name:Bladehold War-Whip
+ManaCost:1 R W
+Types:Artifact Equipment
+K:For Mirrodin
+S:Mode$ ReduceCost | ValidCard$ Equipment.Other | ValidSpell$ Activated.Equip | Activator$ You | Amount$ 1 | Description$ Equip abilities you activate of other Equipment cost {1} less to activate.
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Double Strike | Description$ Equipped creature has double strike.
+K:Equip:3 R W
+DeckHas:Ability$Token & Type$Rebel & Color$Red
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/bladehold_war_whip.txt:8]: DeckHint for more Equipment?

File: forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt [hunk 10]
```diff
@@ -0,0 +1,11 @@
+Name:Rhuk, Hexgold Nabber
+ManaCost:2 R
+Types:Legendary Creature Goblin Rebel
+PT:2/2
+K:Trample
+K:Haste
+T:Mode$ Attacks | ValidCard$ Creature.equipped+Other | Execute$ TrigAttach | TriggerDescription$ Whenever an equipped creature you control other than CARDNAME attacks or dies, you may attach all Equipment attached to that creature to NICKNAME.
+T:Mode$ ChangesZone | ValidCard$ Creature.equipped+Other | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigAttach | Secondary$ True | TriggerDescription$ Whenever an equipped creature you control other than CARDNAME attacks or dies, you may attach all Equipment attached to that creature to NICKNAME.
+SVar:TrigAttach:DB$ Attach | Object$ AttachedTo TriggeredCard.Equipment | Defined$ Self | Optional$ True
```
[the remark this answers is not in the document]
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/rhuk_hexgold_nabber.txt:9]: hm while i don't like OptionalDecider, this seems like the better way

**Review by tool4ever — APPROVED** [review]

**Pull request merged**